src/main/java/io/javalin/core/util/Util.kt

```diff
@@ -68,6 +79,11 @@ object Util {
             |compile "${dependency.groupId}:${dependency.artifactId}:${dependency.version}"
             |-------------------------------------------------------------------""".trimMargin()
 
+    private fun missingDependencyMessageOptional() = """|
```

Comment:
This shouldn't be required, as the message should only be displayed if the user has actively enabled the feature. There should be something like `swaggerOptions.enableSchemaValidation()` where you can do the dependency check.

(If you want to make general changes to the Util, please do it in a separate commit.)